benchmark/src/jmh/kotlin/kotlinx/benchmarks/json/TwitterFeedStreamBenchmark.kt

```diff
@@ -0,0 +1,84 @@
+package kotlinx.benchmarks.json
+
+import com.fasterxml.jackson.databind.DeserializationFeature
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.module.kotlin.jacksonObjectMapper
+import kotlinx.benchmarks.model.MacroTwitterFeed
+import kotlinx.benchmarks.model.MicroTwitterFeed
+import kotlinx.serialization.json.*
+import org.openjdk.jmh.annotations.*
+import java.nio.file.Files
+import java.nio.file.Path
+import java.util.concurrent.TimeUnit
+import kotlin.io.path.deleteIfExists
+import kotlin.io.path.outputStream
+
+@Warmup(iterations = 7, time = 1)
+@Measurement(iterations = 7, time = 1)
+@BenchmarkMode(Mode.Throughput)
+@OutputTimeUnit(TimeUnit.SECONDS)
+@State(Scope.Benchmark)
+@Fork(2)
+open class TwitterFeedStreamBenchmark {
+    val resource = TwitterFeedBenchmark::class.java.getResource("/twitter_macro.json")!!
+    private val twitter = Json.decodeFromString(MacroTwitterFeed.serializer(), resource.readText())
+
+    private val jsonIgnoreUnknwn = Json { ignoreUnknownKeys = true }
+    private val objectMapper: ObjectMapper =
+        jacksonObjectMapper().configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false)
+
+
+    private var file: Path? = null
+
+    @Setup
+    fun init() {
+        file = Files.createTempFile("json_benchmark", "tmp")
```

Comment:
[nit] typically, for such benchmarks I prefer measurement from the stream that is already presented in the memory (i.e. BAIS/BAOS).

While it definitely not represent the "real"-ish workload, it does not skew our measurements and we can be sure that e.g. the benchmark score is not affected by the quality and/or load of the disk or file system,
In the end, the goal of this benchmark is to get precise (for this particular data) comparison with Jackson and we do not want our differences to be obscured or hidden by 3-rd party factors